integrations/instructor-embedder.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: integration
+name: INSTRUCTOR Embedders
+description: A component for computing embeddings using INSTRUCTOR embedding models - built for Haystack 2.0.
+authors:
+    - name: Ashwin Mathur
+      socials:
+        github: awinml
+        twitter: awinml
+        linkedin: ashwin-mathur-ds
+    - name: Varun Mathur
+      socials:
+        github: vrunm
+        twitter: vrunmnlp
+        linkedin: varun-mathur-ds
+pypi: https://pypi.org/project/instructor-embedders-haystack/
+repo: https://github.com/deepset-ai/haystack-extras/tree/main/components/instructor-embedders
+type: Custom Node
+report_issue: https://github.com/deepset-ai/haystack-extras/issues
+version: Haystack 2.0
+---
+
+[![PyPI - Version](https://img.shields.io/pypi/v/instructor-embedders-haystack.svg)](https://pypi.org/project/instructor-embedders-haystack) &nbsp;&nbsp;&nbsp;[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/instructor-embedders-haystack.svg)](https://pypi.org/project/instructor-embedders-haystack)
```

Comment:
I think there are some backspace conversions here that have to be removed :) probably a copy-paste issue. Also present below. there are a lot of `&nbsp` tags. Other than that looks good 👍 🚀 - let's remove those and I'll merge

Reply:
I added the `&nbsp` tags for extra spacing, though in this case I can remove them and use markdown formatting instead. I will remove them and make the necessary changes. Thanks!